From b5a248f4ba99988e5a34cd510219d6c9ee55977b Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Fri, 12 Apr 2019 02:10:12 +0300
Subject: [PATCH 3/3] build: Include AudioPackage14.mk instead of AllAudio.mk

* AllAudio.mk doesn't contain latest material files and also
  contains multiple duplicate files. Just copy the latest
  material ones and we'll include the older files in vendor/lineage

Change-Id: I7afdd9b94cc8148fa4ca58fe28310215aed8d27b
---
 target/product/full_base.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/full_base.mk b/target/product/full_base.mk
index 4f143ec23..cfc51a999 100644
--- a/target/product/full_base.mk
+++ b/target/product/full_base.mk
@@ -41,7 +41,7 @@ PRODUCT_PACKAGES += \
 PRODUCT_LOCALES := en_US
 
 # Get some sounds
-$(call inherit-product-if-exists, frameworks/base/data/sounds/AllAudio.mk)
+$(call inherit-product-if-exists, frameworks/base/data/sounds/AudioPackage14.mk)
 
 # Get a list of languages.
 $(call inherit-product, $(SRC_TARGET_DIR)/product/languages_full.mk)
-- 
2.17.1

